scss/book-list.scss

```diff
@@ -0,0 +1,33 @@
+body {
+  padding-top: 70px;
+}
+
+.book-item {
+  max-width: 198px;
+  padding: 28px 28px 22px;
+
+  .book-cover {
+    width: 142px;
+  }
+
+  .book-title {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  .book-author {
+    color: $equalizer-grey;
+    font-size: 14px;
+  }
+
+  > * + * {
```

Comment:
![image](https://user-images.githubusercontent.com/24418636/109689429-0a9dec80-7b64-11eb-9e1b-44c6d21b1674.png)

Maybe the `nth-child` is better than this 😝